### PR TITLE
add number of extensions

### PR DESCRIPTION
**Pull request by rkimera94**
Introduce the number of extensions on the list of file extensions.  @breck7

### CODE REVIEW

File: code/SiteBuilder.ts [hunk 1]
```diff
@@ -50,25 +50,25 @@ const buildImportsFile = (filepath, varMap) => {
...
+          return `replace ${key}
  pipeTable
   ${new TreeNode(value.rows)
-    .toDelimited("|", value.header, false)
-    .replace(/\n/g, "\n  ")}`
+              .toDelimited("|", value.header, false)
+              .replace(/\n/g, "\n  ")}`
 
-          value = value.toString()
+        value = value.toString()
```
Comment (breck7) [inline code/SiteBuilder.ts:64]: we are probably missing some prettier config settings. not sure why all these lines were changed

**Comment by breck7** [comment]
nice work @rkimera94 !

this is live now.

really interesting! that's some cool signal:

Screen Shot 2022-09-28 at 5 08 51 AM

File: code/SiteBuilder.ts (hunk 1 above)
Comment (rkimera94) [reply code/SiteBuilder.ts:64]: yes, that's why the files changed.

**Comment by celtic-coder** [comment]
Yes, good work, Kimera (@rkimera94)! 🙂

**Comment by rkimera94** [comment]
Thanks, @celtic-coder

**Pull request merged**